clarity/src/vm/docs/mod.rs

```diff
@@ -1401,6 +1401,23 @@ The signature includes 64 bytes plus an optional additional recovery id (00..03)
  0x03adb8de4bfb65db2cfd6120d55c6526ae9c52e675db7e47308636534ba7786110) ;; Returns false"
 };
 
+const SECP256R1VERIFY_API: SpecialAPI = SpecialAPI {
+    input_type: "(buff 32), (buff 64), (buff 33)",
+    snippet: "secp256r1-verify ${1:message-hash} ${2:signature} ${3:public-key})",
+    output_type: "bool",
+    signature: "(secp256r1-verify message-hash signature public-key)",
+    description: "The `secp256r1-verify` function verifies that the provided signature of the message-hash
+was signed with the private key that generated the public key.
+`message-hash` is typically the `sha256` of a message and `signature` is the raw 64-byte signature.
+Note that this is NOT the Bitcoin (or default Stacks) signature scheme, secp256k1, but rather the
+NIST P-256 curve (also known as secp256r1).",
```

Comment:
It also occurs to me that we might want to mention that this function takes high-S signatures, whereas the secp256k1-verify function does not.